packages/module/patternfly-docs/content/examples/verticalTab.css

```diff
@@ -32,22 +43,23 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
-.ws-react-e-vertical-tabs .vertical-tabs-pf-tab.active > a {
-  color: var(--pf-t--global--text--color--brand--clicked);
+.ws-react-e-vertical-tabs .vertical-tabs-pf-tab > a.disabled {
```

Comment:
I didn't see any examples of a disabled link in this extension, but it was in figma, so added support for `<a class="disabled">`